build/teamcity/cockroach/ci/tests/local_roachtest_fips_impl.sh

```diff
@@ -16,6 +16,11 @@ bazel build --config=$CROSSLINUX_CONFIG --config=ci //pkg/cmd/cockroach-short \
       //pkg/cmd/roachprod \
       //pkg/cmd/workload
 
+bazel build --config=$CROSSLINUX_CONFIG --config=ci --config=force_build_cdeps //c-deps:libgeos
+
+cp _bazel/bin/c-deps/libgeos_foreign/lib/libgeos.so lib/libgeos.so
```

Comment:
Use `BAZEL_BIN` from below instead of hard-coding `_bazel/bin`. Also, you'll want to `chmod a+w` the libraries after copying them.

Reply:
Thanks! Seems like it worked now, let me know if there's anything else I should be doing differently.